app/controllers/analytics/districts_controller.rb

```diff
@@ -51,6 +51,22 @@ def patient_list
     )
   end
 
+  def patient_list_with_history
+    recipient_email = current_admin.email
+
+    PatientListDownloadJob.perform_later(recipient_email, "district", {
+      district_name: @organization_district.district_name,
+      organization_id: @organization_district.organization.id
+    }, with_medication_history: true)
+
+    redirect_to(
+      analytics_organization_district_path(id: @organization_district.district_name),
+      notice: I18n.t("patient_list_email.notice",
+        model_type: "district",
+        model_name: @organization_district.district_name)
+    )
+  end
```

Comment:
Could we re-use the existing route with a param, instead of a new route?

Reply:
I slightly prefer the separate actions, because we're going to keep both line lists around and we could expect these to diverge over time. Couldn't think of a strong reason to merge them in one, wdyt?

Reply:
My thought was, this is the beginning of various "flavors" of a line list. Imagine a more sophisticated mechanism to say "I want blood sugars, I don't want medication history, I want regn info", etc. However, if we do that, it'll probably live somewhere brand new anyway.

Agree, let's avoid early optimization and keep them separate 👍